src/components/index.tests.tsx

```diff
@@ -13,8 +16,26 @@ const demoQuestions = [
 ]
 
 // Localized test
-class MyTest extends React.Component<any, any> {
+class FaqTest extends React.Component<any, any> {
   render() {
     return <Faq questions={demoQuestions} />
   }
 }
+
+class CheckboxInputTest extends React.Component<any, any> {
+  render() {
+    return (
+      <>
+        <CheckboxInput
+          name="name: string -- is required"
```

Comment:
My main goals on these iterations are:

1. stronger typing
2. allow for optional props e.g. `foo?: TYPE`

I noticed with the Nora prototyping stuff that some of the EDS components in that env were requiring all props to be supplied even if with `prop={undefined}` which  is silly.

So, my thought is testing with only required props helps to validate that here